meson_options.txt

```diff
@@ -2,3 +2,5 @@ option('static-linkage', type : 'boolean', value : false,
   description : 'Create statically linked binaries.')
 option('magic-install-prefix', type : 'string', value : '',
   description : 'Prefix where libmagic has been installed')
+option('with-zimwriterfs', type : 'boolean', value : false,
```

Comment:
Please turn it on directly in this commit not in the next one.
It is useless (but "pollute" the git history) to do this in two steps.

Reply:
optionality of zimwriterfs removed according to https://github.com/openzim/zim-tools/pull/104#pullrequestreview-436483298